backend/models/generic.go

```diff
@@ -1,13 +1,11 @@
 package models
 
 import (
-	"gorm.io/gorm"
 	"time"
 )
 
 type Common struct {
-	ID        uint           `gorm:"primaryKey" json:"id"`
-	CreatedAt time.Time      `json:"created_at"`
-	UpdatedAt time.Time      `json:"updated_at"`
-	DeletedAt gorm.DeletedAt `gorm:"index" json:"deleted_at"`
+	Id        uint      `json:"id"`
```

Comment:
why is id no longer a primary key?

Reply:
It still is, you don't need to specify `gorm:"primaryKey"` if the field name is Id: https://gorm.io/docs/models.html#Conventions

Reply:
figured was case sensitive, makes sense